src/components/WhaleInfo/WhaleInfoSidebar.tsx

```diff
@@ -4,14 +4,14 @@ import "./WhaleInfoSidebar.scss";
 export function WhaleInfoSidebar() {
```

Comment:
Can we change the syntax here for consistency?
```suggestion
export const WhaleInfoSidebar: React.FC = () => {
```